### DISCUSSION TITLE
Not able to change splash screen

### DISCUSSION THREAD

**Discussion by KrishnaSangaleKD** [nylo-core/nylo#148] [Q&A]
I'm building Android and IOS application using Nylo flutter and I have to add customizable splash screen to my app, I have tried too many ways to implement it but it's not working and using same way it's working on plane template of flutter

**Comment by agordn52** [comment]
Hi @KrishnaSangaleKD,

Have you tried https://pub.dev/packages/flutter_native_splash?

**Reply by KrishnaSangaleKD** [reply]
Yes, I've tried it but it's not working

**Reply by agordn52** [reply]
I've just tried this and it's working.

Please ensure that you've read their documentation [here](https://pub.dev/packages/flutter_native_splash) 👍

![nylo-splash](https://github.com/nylo-core/nylo/assets/17294994/16cf6ad9-276e-426f-bf6c-b062bcdf9858)
b.dev/packages/flutter_native_splash)

**Reply by KrishnaSangaleKD** [reply]
Please correct me if I've written anything wrong
flutter_native_splash:
  android: true
  ios: true
  color: "#30acff"
  image: "assets/netflix.png" (using this image for testing purpose)
  
  this is my code in pubspec.yaml file
  
  dart run flutter_native_splash:create
  then run this command and it will generate splash screen 
 
![image](https://github.com/nylo-core/nylo/assets/101379539/5ef1f085-4b31-4e2f-af2d-0439d03ea544)

i'm checking the result on actual android device it's not working still it's showing default nylo splash screen
